common/authorization/oauthAuthorizer.go

```diff
@@ -137,34 +140,10 @@ func (a *oauthAuthority) parseToken(tokenStr string, verifier jwt.Verifier) (*JW
 
 func (a *oauthAuthority) validateTTL(claims *JWTClaims) error {
 	if claims.TTL > a.authorizationCfg.MaxJwtTTL {
-		return fmt.Errorf("TTL in token is larger than MaxTTL allowed")
+		return fmt.Errorf("token TTL: %d is larger than MaxTTL allowed: %d", claims.TTL, a.authorizationCfg.MaxJwtTTL)
 	}
 	if claims.Iat+claims.TTL < time.Now().Unix() {
-		return fmt.Errorf("JWT has expired")
+		return errors.New("JWT has expired")
```

Comment:
Remark: No need to do anything

We probably need to pull these out as distinct error types such that folks can match against them rather than string matching. Anyway, a change for another PR

Reply:
I was thinking about that, but the default behaviour now is to return `DecisionDeny` without any explanation. You can see this error in server logs with logging level set to `debug`